src/components/PGN.vue

```diff
@@ -201,6 +210,10 @@ export default {
         commentsObj[el.fen] = el.comment;
       });
       return commentsObj;
+    },
+    flipBoard() {
+      this.isBoardFlipped = !this.isBoardFlipped;
+      this.loadPgn();
```

Comment:
This is a reminder for myself

@TerabyteTiger create a new issue so that flipping the board doesn't reset the game state